Bottom: 6e9336f1a86e4c94bcac3d230ce0062563ea002c
Top:    bbccf1ab501b389808360ffd5f18ace2c111d753
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-09-05 23:38:51 +0100

Refresh of hid-gtk-connect-up-the-file-ch

---

diff --git a/src/hid/gtk/gui-top-window.c b/src/hid/gtk/gui-top-window.c
index 7426bf3..a9b2c5e 100644
--- a/src/hid/gtk/gui-top-window.c
+++ b/src/hid/gtk/gui-top-window.c
@@ -367,6 +367,8 @@ connect_file_change_monitor (GhidGui *_gui)
   /* Ensure any existing monitor is disconnected */
   disconnect_file_change_monitor (_gui);
 
+  printf ("PCB Filename is %s\n", PCB->Filename);
+
   if (PCB->Filename == NULL ||
       *PCB->Filename == '\0')
     return;
@@ -415,6 +417,7 @@ ghid_notify_save_pcb (const char *file, bool done)
    * and this notification causes us to disconnect our old file-monitor
    * (pointing at the old file), then we re-connect to the new file.
    */
+  printf ("Filename is %s, PCB->Filename is %s\n", file, PCB->Filename);
   if (strcmp (file, PCB->Filename) != 0)
     return;
